dpe/src/dpe_instance.rs

```diff
@@ -385,10 +390,11 @@ impl<C: Crypto> DpeInstance<'_, C> {
             .position(|context| context.state == ContextState::Inactive)
     }
 
-    fn get_default_context(&mut self) -> Option<&mut Context> {
+    fn _get_default_context(&mut self, locality: u32) -> Option<&mut Context> {
         self.contexts.iter_mut().find(|context| {
             matches!(context.state, ContextState::Active | ContextState::Locked)
                 && context.handle == Self::DEFAULT_CONTEXT_HANDLE
+                && context.locality == locality
         })
     }
```

Comment:
Is this function needed? Seems like in most cases we would just call `get_active_context_pos`

Reply:
Yeah, its pretty useless especially because I'm going to be taking out the `ContextType::Default` and `ContextState::Locked` in a PR soon.